docs/site/decorators/Decorators_authorize.md

```diff
@@ -0,0 +1,87 @@
+---
+lang: en
+title: 'Authorization Decorators'
+keywords:
+  LoopBack 4.0, LoopBack, Node.js, TypeScript, OpenAPI, Decorator, Authorization
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Decorators_authorize.html
+---
+
+## Authorization Decorator
+
+Syntax:
+
+- `@authorize({resource: 'order', scopes: ['create']})`
+
+The authorization decorator is used to provide access control metadata. As part
+of the component [`@loopback/authorization`](../Authorization-component.md), it
+is applied to controller members and specifies who can perform which operations
+to the protected resource.
+
+`@authorize` takes in an object in type `AuthorizationMetadata`. The syntax
+example specifies the `resource` and `scopes`. A full list of the available
+configuration properties are:
+
+`allowedRoles`/`deniedRoles`: Define the ACL based roles. It should be an array
```

Comment:
i think you might mean to add bullets to split up each item?

Reply:
right